### PR TITLE
(new registry) HMS Research Computing

### PR DESCRIPTION
**Pull request by vladdoster**
- Add metadata markdown file
- Add robot image

### CODE REVIEW

**Comment by vsoch** [comment]
Looks like we need an update to the circle deployment (it is in fact the original version 1!) I'll get this in soon, then we can update the branch here and run tests/preview.

**Comment by vladdoster** [comment]
> Looks like we need an update to the circle deployment (it is in fact the original version 1!) I'll get this in soon, then we can update the branch here and run tests/preview.

I just proposed a PR to move CI deployment to a GH action.

**Comment by vsoch** [comment]
hey @vladdoster ! The issue with Circle is fixed, and I think if you grab the latest commit from master (with this update https://github.com/singularityhub/containers/pull/12) it should be able to run and we can preview your site.

**Comment by vladdoster** [comment]
mhm... I'm not sure how to get your changes out of the branch?

File: _registries/hms-rc_registry.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: registry
+title:  "HMS Research Computing Container Registry"
+base: https://sregistry.hms.harvard.edu
+date:   2021-01-16 13:19:12
+author: vladdoster
+categories:
+- registry
+img: robots/hms_rc-robot.png
```
Comment (vsoch) [inline _registries/hms-rc_registry.md]: Looks like there is one issue with showing your robot image, I'd suggest we try making this `hms-rc_robot.png`
```suggestion
img: robots/hms-rc_robot.png
```

File: _registries/hms-rc_registry.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+layout: registry
+title:  "HMS Research Computing Container Registry"
+base: https://sregistry.hms.harvard.edu
+date:   2021-01-16 13:19:12
+author: vladdoster
+categories:
+- registry
+img: robots/hms_rc-robot.png
+thumb: robots/hms_rc-robot.png
```
Comment (vsoch) [inline _registries/hms-rc_registry.md]:
```suggestion
thumb: robots/hms-rc_robot.png
```

**Comment by vsoch** [comment]
You had it done correctly before - if you pulled the commit from the master branch, my commit and changes _should_ be there. I'm not sure what you did now, but I no longer see my commit.

Update: it looks like the issue is that you force pushed (and essentially erased this commit) https://github.com/singularityhub/containers/commit/35b61b8608c4235a50c792747cbee6699eac245d.

**Comment by vsoch** [comment]
I'm not sure what your git history looks like, but if you want to have it done cleanly you can checkout a new branch from the current master (make sure to pull from the upstream master to your branch first) and then just add the robot image and metadata file. I am going for a run but I'll be back later!

**Comment by vsoch** [comment]
Haha I'm still here! Here is the index preview:

- https://47-101526766-gh.circle-artifacts.com/0/containers/index.html

And your page.

- https://47-101526766-gh.circle-artifacts.com/0/containers/registry/hms-rc_registry/index.html

We likely need to remove the listing of containers (it's not really reasonable anymore) but the PR is otherwise good! And I realize I'm going to squash and merge so the git history doesn't matter so much.

**Pull request merged**